### PR TITLE
feat: Implement intrinsics _mm_div_sd

### PR DESCRIPTION
**Pull request by HowJMay**

### CODE REVIEW

**Comment by HowJMay** [comment]
depends on #269

File: sse2neon.h [hunk 1]
```diff
@@ -3496,6 +3496,16 @@ FORCE_INLINE __m128d _mm_div_pd(__m128d a, __m128d b)
...
 }
 
+// Divide the lower double-precision (64-bit) floating-point element in a by the
+// lower double-precision (64-bit) floating-point element in b, store the result
+// in the lower element of dst, and copy the upper element from a to the upper
+// element of dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_div_sd
+FORCE_INLINE __m128d _mm_div_sd(__m128d a, __m128d b)
+{
+    return _mm_move_sd(a, _mm_div_pd(a, b));
```
Comment (jserv) [inline sse2neon.h:3506]: This implementation might be slightly heavier. For Aarch64, you can reimplement `_mm_div_sd` as following:
```c
float64x2_t tmp = vdivq_f64(vreinterpretq_f64_m128d(a), vreinterpretq_f64_m128d(b));
return vreinterpretq_m128d_f64(vsetq_lane_f64(vgetq_lane(vreinterpretq_f64_m128d(a), 1), tmp, 1));
```

Less instructions.

**Review by jserv — APPROVED** [review]

File: sse2neon.h (hunk 1 above)
Comment (marktwtn) [reply sse2neon.h:3582]: @jserv I wonder what if I change the implementation of `_mm_move_sd` from `vcombine + vget_low/high_f32` to `vext_f32`?
Then @HowJMay can use `_mm_move_sd` without modification.

Comment (jserv) [reply sse2neon.h:3582]: Agree. Let's close this pull request and concentrate on `_mm_move_sd` changes.

Comment (marktwtn) [reply sse2neon.h:3582]: I tried to implement `_mm_move_sd` with `vext_f32` and found out that it needs at least two `vext_f32` to do the same thing.
I think we should leave `_mm_move_sd` unchanged.

**Review by marktwtn — APPROVED** [review]

**Pull request merged**